projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/kubernetes/DataJobsKubernetesService.java

```diff
@@ -24,9 +24,8 @@ public class DataJobsKubernetesService extends KubernetesService {
 
   public DataJobsKubernetesService(
       @Value("${datajobs.deployment.k8s.namespace:}") String namespace,
-      @Value("${datajobs.deployment.k8s.kubeconfig:}") String kubeconfig,
-      @Value("${datajobs.control.k8s.k8sSupportsV1CronJob}") boolean k8sSupportsV1CronJob) {
-    super(namespace, kubeconfig, k8sSupportsV1CronJob, log);
+      @Value("${datajobs.deployment.k8s.kubeconfig:}") String kubeconfig) {
+    super(namespace, kubeconfig, log);
     if (StringUtils.isBlank(kubeconfig) && !new File(kubeconfig).isFile()) {
```

Comment:
please provide to PR description the Kubernetes compatibility expected with the change proposed, before and after

Reply:
Could you provide the Kuberenetes versions supported prior/after the change? It is still not clear to me...

Reply:
In addition, I understand the defaults need to be switched to v1 in order to use >=1.25. 
So, why not changing the defaults and still keep the backwards compatibility?

Reply:
Yeah sorry I see where the confusion is.
We won't be able to use the latest versions of the k8s client SDK. not the latest version of the k8s cluster. 
Does that make sense?

Reply:
What's the latest k8s client backwards compatibility, could you please link to the PR?